ci-operator/jobs/openshift/cluster-api-provider-agent/openshift-cluster-api-provider-agent-master-periodics.yaml

```
@@ -1,4 +1,76 @@
 periodics:
+- agent: kubernetes
+  cluster: build03
+  cron: 59 3 * * *
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: cluster-api-provider-agent
+  labels:
+    ci-operator.openshift.io/cloud: packet
+    ci-operator.openshift.io/cloud-cluster-profile: packet-assisted
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-openshift-cluster-api-provider-agent-master-e2e-capi-provider-hypershift-periodic
+  spec:
```

I'll suggest adding notifications in case of failures:
```suggestion
  reporter_config:
    slack:
      channel: '#assisted-deployment-ci'
      job_states_to_report:
      - failure
      - error
      report_template: '{{ if eq .Status.State "success" }} :green_jenkins_circle:
        {{ else }} :red_jenkins_circle: {{ end }} Job *{{.Spec.Job}}* ended with *{{.Status.State}}*.
        <{{.Status.URL}}|View logs>'
  spec:
```
same for other jobs here

---

This is auto-generated.

---

I know
But we couldn't make it auto-generate the slack notifications

---

OK, I'd like to have the slack notification for the periodic job.
I don't think I want it for presubmits.
Unsure about postsubmits.

---

I meant other periodic jobs
We usually have notifications for periodics only